src/common/options.cc

```diff
@@ -8362,6 +8362,12 @@ std::vector<Option> get_mds_client_options() {
     .set_description("pass default_permisions to FUSE on mount")
     .set_flag(Option::FLAG_STARTUP),
 
+#ifdef CEPH_HAVE_SPLICE
+    Option("fuse_read_splice", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(true)
+    .set_description("enable splice to reduce the memory copies"),
+#endif
+
```

Comment:
Why add this tunable? Do you envision problems that may require someone to disable it, or is this just leftover from performance testing?

Reply:
Locally from my local testings and compared to using fuse_reply_iov case, the splice read just improved around 1~10%. 

So just want to give one simple way to try the fuse_reply_iov case in some case ?